h11/tests/test_connection.py

```diff
@@ -59,7 +59,7 @@ def req(cl=None, te=False):
             assert _body_framing(meth, r) == ("content-length", (0,))
 
     # Transfer-encoding
-    for kwargs in [{"te": True}, {"cl": 100, "te": True}]:
```

Comment:
Well, it's testing that `_body_framing` correctly ignored the `Content-Length` when `Transfer-Encoding` was provided :-). We don't necessarily need to test it right here in this way, but it is a useful thing to test.